### DISCUSSION TITLE
[Feature] Download asset directly from the timeline

### DISCUSSION THREAD

**Discussion by ryanrpj** [immich-app/immich#12977] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Some video files have such high bitrate that it bottlenecks my local network when opened. This problem is even worse on mobile, where it may cause lag in the app while buffering. It is always a bad experience when it comes to downloading said files, because I have no other option other than to first open the file, and then start to download it.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
For web, this is already supported. For mobile, this is a duplicate of #5068.

Please search for duplicates before opening a thread.
